.clang-format

```diff
@@ -231,7 +231,7 @@ SpacesInLineCommentPrefix:
 #   InEmptyParentheses: false
 #   Other: false
 # SpacesInSquareBrackets: false
-Standard: c++17
+Standard: c++20
```

Comment:
I was curious and I checked the [clang-format docs](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#standard) if `c++23` existed and it doesn't seem so.